pages/index.html

```diff
@@ -38,6 +38,9 @@
                         getValues: function() {
                             return attachedIds;
                         }
+                    },
+                    UserActions: {
+                        userId: 'test'
```

Comment:
This too perhaps?

Reply:
Require to test stuff 😄

Reply:
Oh i forget is this file just for running the dev server?

Reply:
Yeah 😄

Reply:
oups!